### DISCUSSION TITLE
Can I upgrade from version v1.0.11.9 to newest one?

### DISCUSSION THREAD

**Discussion by johnflint1985** [TumblThreeApp/TumblThree#318] [Q&A (Question & Answer)]
Hello.
I am using a version v1.0.11.9 and all of a sudden I cannot authenticate myself into the program? Is there any solution to this and if I migrate to the newest version, can I use the old files from Index folder? I have there quite a library and want to keep it. 
Thank you

**Comment by thomas694** [comment]
Hello John,

that's quite an old version.
You have just been chosen to serve as a guinea pig. 🙂 

As always make yourself backups of your installation folder, the Index folder, your Blogs folder and settings folder (%LOCALAPPDATA%\TumblThree). In general you should have your Blogs folder outside from your installation folder (you could move it to another location and adjust the path in the settings).
Then empty your installation folder and extract the new version into it, or just use a new installation folder (in case of portable mode you would need to keep/copy the settings files).
Afterwards start the new version and wait a moment, or two.
Good luck!

**Comment by johnflint1985** [comment]
Thank you. Let me try that and see the results.

**Comment by thomas694** [comment]
Hello,
did you have success with it?

**Comment by johnflint1985** [comment]
Hello. I tried and what I noticed is that even with an older version once
you register and authorize yourself, it will still say that you are NOT
authorized. But in fact, you are because it allows you to see and work with
all SVC downloads. So it is some sort of a bug, that doesn't recognize the
authorization if it is done. For the time being it works and I don't want
to migrate to the new version since I need to put in hundreds of tumble
blog names that are already there. Since it doesn't recognize the old files
from my version to the new one you can't simply copy and paste old Index
folder with all its contents into the new version Blog folder. Thus I
decided to stay with an old one for now. But in the future I need to go to
the new version.

thank you for a follow up
Alex

**Comment by thomas694** [comment]
> Since it doesn't recognize the old files
from my version to the new one you can't simply copy and paste old Index
folder with all its contents into the new version Blog folder. 

I'm not sure why you think it cannot. It should convert the old blog files to the new format.
But you can try the new version with your current blogs without interfering with your old installation:
- unzip the new version to a new folder somewhere
- create a `Settings.json` file inside the new version's folder with content (that way it doesn't even touch your old settings in `%LOCALAPPDATA%\TumblThree\Settings`):
```
{
  "PortableMode": true
}
```
- start (the new) TumblThree once and choose a new download location or keep the default (subfolder Blogs) and close it. You can also skip this step and just create a subfolder Blogs by hand.
- copy your Index folder from your old version into the download location of your new version
- start and test the new TumblThree version

**Comment by johnflint1985** [comment]
Thanks for the tip! I should try that as well.

**Comment by thomas694** [comment]
The issue has been closed. You can still comment. Feel free to ask for reopening the issue if needed.

**Comment by johnflint1985** [comment]
Awesome, thanks!
